### ISSUE TITLE
API is out of sync with the unity implementation

### ISSUE THREAD

**Issue by shadowmint** [loodakrawa/SpriterDotNet#10]
The implementations of `ApplyTransform()` and `Animate()` are no longer correct due to changes in the core API.

These are easily fixed, but I got stuck trying to fix the importer, which errors with:

```
Assets/lib/spriter/SpriterDotNetUnity/SpriterImporter.cs(77,26): error CS0246: The type or namespace name `SpriterMainLineKey' could not be found. Are you missing a using directive or an assembly reference?
```

Although the compiled DLL in the unity folder continues to work, attempting to import or compile the SpriterDotNet core no longer does.

Perhaps you could consider tagging/branching the repo at a known working version for unity?

**Comment by loodakrawa** [comment]
The Unity plugin is out of sync due to a massive push to implement all the remaining features in the core branch. I'm actively working on the Unity plugin and the expected release date is November 8th.

**Comment by loodakrawa** [comment]
Fixed in v1.0.0
